models/topics-model.js

```diff
@@ -1,7 +1,16 @@
 const db = require('../db/connection')
+const fs = require('fs')
 
 exports.selectTopics = () => {
     return db.query(`SELECT * FROM topics`).then((data) => {
         return data.rows
     })
+}
+
+exports.getAllEndpoints = () => {
+    return new Promise((resolve, reject) => {
+        fs.readFile('endpoints.json', 'utf8', function(err, data){
+            if (err) reject(err);
+            else resolve(JSON.parse(data));
+    })})
```

Comment:
This isn't strictly necessary, you could just require in your json file - but it does the job!